shared/chat/inbox/row/small-team/bottom-line.js

```diff
@@ -159,53 +114,74 @@ class BottomLine extends PureComponent<Props> {
     } else {
       return null
     }
+
     return (
       <Box
-        style={{
-          ...globalStyles.flexBoxRow,
-          backgroundColor: isMobile ? this.props.backgroundColor : undefined,
-          flexGrow: 1,
-          height: isMobile ? 20 : 17,
-          maxHeight: isMobile ? 20 : 17,
-        }}
+        style={collapseStyles([
+          styles.outerBox,
+          {
+            backgroundColor: isMobile ? this.props.backgroundColor : undefined,
+          },
+        ])}
       >
         {this.props.hasResetUsers && (
-          <Meta title="reset" style={resetStyle} backgroundColor={globalColors.red} />
+          <Meta title="reset" style={styles.alertMeta} backgroundColor={globalColors.red} />
         )}
-        <Box
-          style={{
-            ...globalStyles.flexBoxRow,
-            alignItems: 'center',
-            flexGrow: 1,
-            height: '100%',
-            position: 'relative',
-          }}
-        >
-          <Box
-            style={{
-              ...globalStyles.flexBoxRow,
-              ...globalStyles.fillAbsolute,
-              alignItems: 'center',
-            }}
-          >
-            {content}
-          </Box>
+        {this.props.youNeedToRekey && (
+          <Meta title="rekey needed" style={styles.alertMeta} backgroundColor={globalColors.red} />
+        )}
+        <Box style={styles.innerBox}>
+          <Box style={{...globalStyles.fillAbsolute}}>{content}</Box>
         </Box>
       </Box>
     )
   }
 }
-const resetStyle = platformStyles({
-  common: {
-    alignSelf: 'center',
-    marginRight: 6,
+
+const styles = styleSheetCreate({
+  outerBox: {
+    ...globalStyles.flexBoxRow,
+    flexGrow: 1,
+    height: isMobile ? 20 : 17,
+    maxHeight: isMobile ? 20 : 17,
   },
-  isElectron: {},
-  isMobile: {
-    marginTop: 2,
+  innerBox: {
+    ...globalStyles.flexBoxRow,
+    alignItems: 'center',
+    flexGrow: 1,
+    height: '100%',
+    position: 'relative',
   },
-})
-const styles = styleSheetCreate({
+  rekeyNeededContainer: {
+    alignSelf: 'center',
+    backgroundColor: globalColors.red,
+    borderRadius: 2,
+    paddingLeft: globalMargins.xtiny,
+    paddingRight: globalMargins.xtiny,
+  },
+  rekeyNeededText: platformStyles({
+    common: {
+      color: globalColors.white,
+    },
+    isElectron: {
+      fontSize: 11,
+      lineHeight: '13px',
```

Comment:
this should just be  13, platformstyles does the cast for you

Reply:
that gives a flow error because lineHeight has to be a string

Reply:
that type is wrong then, it should be Dimension